Jenkinsfile

```diff
@@ -107,7 +107,7 @@ def rocmnode(name, body) {
     }
 }
 
-rocmtest clang_debug: rocmnode('cdna') { cmake_build ->
+rocmtest clang_debug: rocmnode('mi100+') { cmake_build ->
```

Comment:
why were vega nodes  removed?

Reply:
they were causing issues in CI for the onnxruntime runs. Something we need to sort out at a later date

Reply:
Because the vega nodes continually fail during `make check` with memory failures.  Something that should be investigated to get more life in CI, but PRs right now are false failing due to this problem.

Reply:
I'll open an issue on this.